app/content-scripts/notifications.js

```diff
@@ -733,6 +733,9 @@ const NotificationsContentScript = (function (win, doc) {
 			];
 			const { name } = request;
 			const reqMsg = request.message;
+			if (!reqMsg) {
+				return false;
+			}
```

Comment:
@zarembsky let's investigate why `request.message` is coming in as `undefined` here.